apps/src/scratch/scratch.js

```diff
@@ -0,0 +1,193 @@
+import AudioEngine from 'scratch-audio';
+import Renderer from 'scratch-render';
+import Storage from 'scratch-storage';
+import VM from 'scratch-vm';
+import Blockly from 'scratch-blocks/dist/vertical.js';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+import { getStore, registerReducers } from '@cdo/apps/redux';
+import * as commonReducers from '@cdo/apps/redux/commonReducers';
+import { singleton as studioApp } from '@cdo/apps/StudioApp';
+import ScratchView from './ScratchView';
+import { scratchDefaultProject } from './scratchDefaultProject';
+
+const Scratch = window.Scratch = window.Scratch || {};
```

Comment:
If this is only needed/used for debugging purposes, it feels like we should make that more explicit. Maybe we can accomplish that by just renaming. Maybe it makes sense to put some sort of accessor of `window.__TestInterface__`